### DISCUSSION TITLE
在配置文件设置了ipv6: true，但是会被混入配置-全局配置中没启用IPv6覆盖，基础配置没开启混入的情况下

### DISCUSSION THREAD

**Discussion by rsgdn** [nikkinikki-org/OpenWrt-nikki#194] [Q&A]
导致启动时会看到启动文件里ipv6: false，而且我观摩一下，暂时就ipv6这个选项会被覆盖，其他的功能我配置文件设置true的，相对应全局配置里功能没开启的话，实际启动文件也是true的。

**Comment by morytyann** [comment]
看下[Wiki](https://github.com/morytyann/OpenWrt-mihomo/wiki#%E5%BF%85%E8%A6%81%E9%85%8D%E7%BD%AE)？

**Reply by rsgdn** [reply]
看到了🙏
